Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM ubuntu:20.04
```

Comment:
Should we use a lighter weight `FROM`?

Reply:
Sure, what base would you like to use?

Reply:
Not an expert but I hear Alpine is always good ... except for Python projects.

Otherwise, we can keep ubuntu.  If the runner is already using that image, then its already pulled and ready to go.

Reply:
I gave alpine a shot, but typos didn't work inside. Here is the Dockerfile if you want to give it a try!

```
FROM alpine:3.13
ARG VERSION=1.0.3
ENV VERSION=${VERSION}
RUN apk update && apk add wget
RUN wget https://github.com/crate-ci/typos/releases/download/v${VERSION}/typos-v${VERSION}-x86_64-unknown-linux-gnu.tar.gz && \
    tar -xzvf typos-v${VERSION}-x86_64-unknown-linux-gnu.tar.gz && \
    mv typos /usr/local/bin
ENTRYPOINT ["/usr/local/bin/typos"]
```

Reply:
Its ok. I need to go modify the Linux builds to include musl statically firt.